worlds/cv64/options.py

```diff
@@ -507,11 +507,11 @@ class WindowColorA(Range):
 
 
 class DeathLink(Choice):
-    """
-    When you die, everyone dies. Of course the reverse is true too.
-    Explosive: Makes received DeathLinks kill you via the Magical Nitro explosion instead of the normal death animation.
-    """
-    display_name = "DeathLink"
+    __doc__ = (DeathLink.__doc__ +
+               "\n\n    Explosive: Makes received death links kill you via the Magical Nitro explosion" +
```

Comment:
```suggestion
               "\n\n    Explosive: Makes received death links kill you via the Magical Nitro explosion " +
```
Please fix this:
![Screenshot 2024-11-11 135139](https://github.com/user-attachments/assets/04ba3a83-32ef-45b0-aa3e-c8a86624f1f0)

Reply:
fixed!